### PR TITLE
Use request body in Post request

### PR DESCRIPTION
**Pull request by narusemotoki**
This library was using url parameter always.
It does not work well with large data. It looks the limit is 7KB.

### CODE REVIEW

**Comment by toshipon** [comment]
LGTM

File: memsource/api.py [hunk 1]
```diff
@@ -48,13 +48,13 @@ def _make_url(self, *args, **kwargs):
     def _get(
             self, path: str, params: dict={}, *, timeout: int=constants.Base.timeout.value
     ) -> str:
-        return self._request(constants.HttpMethod.get, path, None, params, timeout)
+        return self._request(constants.HttpMethod.get, path, None, params, None, timeout)
```
Comment (patatepartie) [inline memsource/api.py]: Is it possible to use named parameters here ? The `_request` method has too many parameters, and it's hard to know which one is which without looking at the declaration (and even so).

File: memsource/api.py [hunk 2]
```diff
@@ -157,13 +166,21 @@ def _request(
...
             timeout: {'When takes over this time in one request, raise timeout': (int, float)}
     ) -> {'Parsed esponse body as JSON': dict}:
-        (url, params_with_token) = self._pre_request(path, params)
+        if params:
+            (url, params) = self._pre_request(path, params)
+        else:
+            (url, data) = self._pre_request(path, data)
+
+        arguments = {}
+        for key, value in [('files', files), ('params', params), ('data', data)]:
```
Comment (patatepartie) [inline memsource/api.py]: Maybe you can simplify with a list comprehension ?
```python
arguments = {
    (key, value) for key, value in [('files', files), ('params', params), ('data', data)] if value is not None
}
```

File: memsource/api.py [hunk 3]
```diff
@@ -109,12 +109,21 @@ def _request_stream(
...
+            data: {'Send request with this data': dict},
             timeout: {'When takes over this time in one request, raise timeout': (int, float)}
     ) -> requests.models.Response:
-        (url, params_with_token) = self._pre_request(path, params)
+        if params:
+            (url, params) = self._pre_request(path, params)
+        else:
+            (url, data) = self._pre_request(path, data)
+
+        arguments = {}
```
Comment (patatepartie) [inline memsource/api.py]: Same as https://github.com/gengo/memsource-wrap/pull/51/files#r69511340

File: memsource/api.py (hunk 2 above)
Comment (narusemotoki) [inline memsource/api.py]: Cool! Thank you!

File: memsource/api.py (hunk 1 above)
Comment (narusemotoki) [inline memsource/api.py]: You are right. I will add names.

**Comment by kelvintaywl** [comment]
maybe it's good time to introduce changelog file for this repository? just wondering :) 
ref: http://keepachangelog.com/

**Comment by narusemotoki** [comment]
@kelvintaywl Nice! Thank you!

**Comment by ninotoshi** [comment]
LGTM

**Pull request merged**